test/bigtable/filter.js

```diff
@@ -592,44 +524,39 @@ describe('Bigtable/Filter', function() {
 
   describe('value', function() {
     it('should set the value regex filter', function(done) {
```

Comment:
There's still a false positive here. I can change the code of filter#value to this, and the test still passes:

```js
Filter.prototype.value = function(value) {
  if (!is.object(value)) {
    value = {
      value: value
    };
  }

  if (value.value) {
    var valueReg = Filter.convertToRegExpString(value.value);

    Mutation.convertToBytes(valueReg); // called, but the return value not given to `this.set`
    this.set('valueRegexFilter', valueReg);
  }

  // ...
};
```